data/Witkowski_1991/metadata.yml

```diff
@@ -0,0 +1,257 @@
+    fertile cone were determined. The number of florets in 20 of the most recent flower
+    heads from the younger plants was counted and relative fruit set and seed set
+    estimated. Masses of 20 seeds from the most recent crop of the 10-year-old plants
+    were determined. Numbers of opened and closed follicles per cone were counted.
+    ^p Cones were burnt to rupture the follicles, immersed in water for 1 min and
+    left to dry for a week to aid seed removal. Each closed follicle contained two
+    seeds. Seeds were classified as aborted, eaten/ decayed, firm or dispersed. The
+    viability of firm seeds was determined by germinating two batches of 30 firm seeds
+    of each age. Some of the older ages were pooled owing to low numbers of seeds
+    available. Seeds were surface sterilised using sodium hypochlorite (5%) for 5min,
+    rinsed and placed in filter paper over vermiculite in petri dishes (150 mm diam.).
+    These were moistened with a fungicid solution (Benlate @, 1 gL-I), and kept in
+    a growth cabinet at 15OC. Germination was observed for 50 days and percentage
+    germination calculated.
+- var_in: fire_to_fruiting
+  unit_in: a
+  trait_name: fire_time_from_fire_to_fruiting
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: replicates_seeds_fruit
+  methods: Fifteen young (10 years) and five old (21 years) individuals of each species
+    were chosen at random in autumn 1988 to study their seed bank dynamics. The numbers
+    of fertile (with follicles) and infertile cones produced each year (determined
+    by stem increments, Lamont 1985) were counted. Ten cones (or the total if < 10)
+    of each age were sampled from each plant. Age to first flowering and age to first
+    fertile cone were determined. The number of florets in 20 of the most recent flower
+    heads from the younger plants was counted and relative fruit set and seed set
+    estimated. Masses of 20 seeds from the most recent crop of the 10-year-old plants
+    were determined. Numbers of opened and closed follicles per cone were counted.
+    ^p Cones were burnt to rupture the follicles, immersed in water for 1 min and
+    left to dry for a week to aid seed removal. Each closed follicle contained two
+    seeds. Seeds were classified as aborted, eaten/ decayed, firm or dispersed. The
+    viability of firm seeds was determined by germinating two batches of 30 firm seeds
+    of each age. Some of the older ages were pooled owing to low numbers of seeds
+    available. Seeds were surface sterilised using sodium hypochlorite (5%) for 5min,
+    rinsed and placed in filter paper over vermiculite in petri dishes (150 mm diam.).
+    These were moistened with a fungicid solution (Benlate @, 1 gL-I), and kept in
+    a growth cabinet at 15OC. Germination was observed for 50 days and percentage
+    germination calculated.
+- var_in: post_fire_recruitment
```

Comment:
Should serotiny be mapped too since they measured that in Table 3 and 4?

Reply:
done.